test/java/org/chromium/net/testing/data/content_length_mismatch.html

```diff
@@ -0,0 +1 @@
+Response that lies about content length.
```

Comment:
Do all of these need to be in a file (versus just a string in the code)?

Reply:
Well, as you can imagine, this is exactly like that with [Cronet](https://chromium.googlesource.com/chromium/src/+/refs/heads/main/components/cronet/testing/test_server/data).

At first sight, it seemed easier to just port this "as is": the content of the files is easy to understand, and there are other tests to be ported relying on this.